faq/what-is-the-default-directory-that-zap-uses/index.html

```diff
@@ -146,8 +146,8 @@ <h2 id="linux">Linux</h2>
 <p><code>~/.ZAP</code></p>
 <h2 id="mac-os">Mac OS</h2>
 <p><code>~/Library/Application Support/ZAP</code></p>
-<h2 id="weekly-releases">Weekly releases</h2>
```

Comment:
Not sure if we have links using this anchor, it will no longer work.

Reply:
Dont know if we have perminant ones, but I've def linked to it in posts in the past...

Reply:
Should be okay, I think:
![image](https://user-images.githubusercontent.com/7570458/141138517-3ab4c0a4-72e3-4ca4-9a09-811bd5a61fe8.png)

Reply:
Was not referring specifically to the site (in any case the page is small so should be easy to see).